src/cloud/docker/docker-containers-cli.md

```diff
@@ -51,83 +50,86 @@ These commands are available in any of the containers.
+{:.procedure}
+To view the cron log:
 
 ```bash
 docker-compose run deploy bash -c "cat /app/var/cron.log"
 ```
 
--  The `setup:cron:run` and `cron:update` commands are not available on Cloud and Docker for Cloud environment
+-  The `setup:cron:run` and `cron:update` commands are not available on Cloud and Docker for Cloud environments
 -  Cron only works with the CLI container to run the `./bin/magento cron:run` command
 
-This container has no extra configuration, however if performance problems from cron runs are present you can use the following snippet in your docker-compose.override.yml
+{:.procedure}
+To run cron jobs manually:
 
-```yaml
-  cron:
-    entrypoint: "bash -c"
+```bash
+docker-compose run cron /usr/local/bin/php bin/magento cron:run
 ```
-This disables the cron container from running automatically.
 
-You can then run the cron manually using the following command.
+### Disable the cron container
 
-```bash
-docker-compose run cron /usr/local/bin/php bin/magento cron:run
+If cron runs cause performance problems, you can prevent the cron container from running automatically by adding following snippet to your `docker-compose.override.yml` file.
+
+```yaml
+cron:
+  entrypoint: "bash -c"
 ```
 
-### Deploy Container
+After disabling the cron container, use docker compose to run cron jobs manually.
 
-#### Container Information
+## Deploy container
 
--  Name: deploy
--  Docker Base Image: [magento/magento-cloud-docker-php](https://hub.docker.com/r/magento/magento-cloud-docker-php)
--  Docker Image Based On: [php](https://hub.docker.com/_/php)
+Container name |Docker base image
+-------- | --------
+deploy | [magento/magento-cloud-docker-php], which is based on the [php] Docker image
 
-This container is used for the deploy process. This mimics Magento Cloud behaviour so testing of the build and deploy process is as close to production as possible.
+This deploy container mimics the Magento Cloud deploy process so that testing the build and deploy process is as close to testing in production as possible.
 
-#### Container Usage
+You can also run build and deploy commands manually from the deploy container. For example, the following command reindexes the Magento store:
 
-This container can be used to interact with the application.
-```
+```bash
 docker-compose run build magento-command index:reindex
 ```
 
-### Node Container
+## Node container
 
-#### Container Information
+Container name |Docker base image
+-------- | --------
+node | [node]
 
--  Name: node
--  Docker Base Image: [node](https://hub.docker.com/_/node)
+The Node container is based on the [official Node Docker image][node]. You can use the container to install NPM dependencies, such as Gulp, or run any Node-based command line tools.
```

Comment:
This sentence capitalized node, but the other container sentences did not capitalize the container name. It's fine to leave it if it refers to a proper name, but just pointing it out in case.